api/handlers/org_handler.go

```diff
@@ -96,8 +97,10 @@ func (h *OrgHandler) orgListHandler(ctx context.Context, logger logr.Logger, aut
 	}
 
 	resp := NewHandlerResponse(http.StatusOK).WithBody(presenter.ForOrgList(orgs, h.apiBaseURL, *r.URL))
-	if !isExpirationValid(authInfo.CertData, h.userCertificateExpirationWarningDuration) {
-		resp = resp.WithHeader("X-Cf-Warnings", "Warning: Client certificate has an unsafe expiry date. Please use a short-lived certificate")
+	notAfter, certParsed := decodePEMNotAfter(authInfo.CertData)
+
+	if !isExpirationValid(notAfter, h.userCertificateExpirationWarningDuration, certParsed) {
+		resp = resp.WithHeader("X-Cf-Warnings", fmt.Sprintf("Warning: Client certificate has an unsafe expiry date (%s). Please use a short-lived certificate less than %s.", notAfter.Format(time.RFC3339), h.userCertificateExpirationWarningDuration))
```

Comment:
Do you have an example of what this might return the user? When we IPMed this one we figured the hardest part would be wording this well. 😅 

If I'm reading this correctly, this will be something like `"... Please use a short-lived certificate less than 168h"`? Since that's user configurable it could be something like `86400s`. I'm wondering if there are any go helpers to make that duration friendlier.

If not, this is definitely an improvement over what we've currently got. 🙂

Reply:
I was digging into the time.ParseDuration function and it looks like it should make this more readable. I believe that we're already using this function to read from the config but I may be wrong on that.

```
time.ParseDuration("86400s") // Output: 24h0m0s
```